src/pages/guidelines/ibm-dot-com-style-models/style-model-list.mdx

```diff
@@ -0,0 +1,199 @@
+---
+title: IBM.com style models
+description: IBM.com style models provide a design framework that defines the foundational and structural elements of IBM.com pages in order to establish consistency and coherency across users' journeys in a scalable way.
+tabs: ['Overview', 'Style model list']
+---
+
+<PageDescription>
+
+The primary user intent should drive the foundational structure of any given page. This ensures consistency. These are the known user intents and associated style models for IBM.com pages.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Using style models</AnchorLink>
+<AnchorLink>User intents and style models</AnchorLink>
+
+</AnchorLinks>
+
+## Using Style Models
+
+1. Identify the design intent that matches that of the page to be designed.
```

Comment:
Design intent or user intent? We refer to it as primary user intent on the previous page

Reply:
User intent. Let me fix this.